tests/build/build.go

```diff
@@ -533,6 +534,32 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to finish")
 			})
 
+			It("should validate HACBS taskrun results", func() {
+				var trRefList []string
+				// List Of Taskruns Expected to skipped
+				skipresult := []string{"configure-build", "init", "buildah", "git-clone", "get-clair-scan", "deprecated-image-check", "clamav-scan", "sast-snyk-check", "hacbs-test-evaluation", "summary"}
```

Comment:
I think it would be more future-proof to have an inverse logic here where we specify only the tasks that we want to check out instead of ones that we want to skip. 
This would prevent unexpected behavior when a new task gets added to the build pipeline.

Reply:
I have added the skip result because there are two task reference names as "sanity-label-check". One for label check and another for optional label check.

Reply:
Thanks!